src/sass/mixins.scss

```diff
@@ -17,9 +17,9 @@ Provides SASS mixins
   $color-text-hover: $text-color;
   $color-text-active: $text-color;
   $color-border-focus: darken($button-color, $active-darken);
-  $color-background-disabled: $MW_COLOR_NEUTRAL_4;
-  $color-border-disabled: $MW_COLOR_NEUTRAL_4;
-  $color-text-disabled: $MW_COLOR_NEUTRAL_6;
+  $color-background-disabled: $MW_COLOR_GRAY_4;
+  $color-border-disabled: $MW_COLOR_GRAY_4;
```

Comment:
Shouldn't you use mw-disabled here?

Reply:
Indeed. What about `$color-text-disabled`? What variable to use?

![image](https://user-images.githubusercontent.com/24867662/161260669-321d250c-b75d-473c-9722-d8d0cabc95aa.png)